### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by luisremis** [zsdonghao/u-net-brain-tumor#4]
I was wondering if you would share a pre-trained model. 
I would need to run inference-only, and training the model is taking longer than expected.

Thanks for sharing this project!

**Comment by vibhugarg123** [comment]
Yes, please share pre-trained model and the way it can be used to test data
